javatests/arcs/core/data/ParticleSpecTest.kt

```diff
@@ -48,6 +69,65 @@ class ParticleSpecTest {
     assertFailsWith<IllegalArgumentException> { spec.dataflowType }
```

Comment:
should we have a test case where `spec.dataflowType` doesn't throw?

Reply:
Yes, though it's not shown well in the diff.

Reply:
(to clarify: there are tests like this that exist).